packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -81,26 +94,41 @@ export class Popover implements ComponentInterface {
     this.create();
   }
 
-  // Private methods
+  /**
+   * Emits when an element which is not part of the popover is clicked.
+   * Should be used if you control the state of the popover.
+   */
+  @Event() clickOutside: EventEmitter<void>;
 
   private create() {
-    this.tooltipInstance?.destroy();
+    this.tippyInstance?.destroy();
 
     const target = this.inoFor
       ? document.getElementById(this.inoFor)
       : this.el.parentElement;
 
-    const options = {
+    if (!target && this.inoFor) {
+      console.warn(
+        `The element with the id '${this.inoFor}' could not be found.`
+      );
+    }
+
```

Comment:
Maybe also a good moment to resolve the jumping content problem

```ts
// Add the display property on creation to avoid jumping content
// see https://atomiks.github.io/tippyjs/v6/html-content/#element
this.el.style.display = 'block';
```

and 

```css
ino-popover {
  display: none;
}